### PR TITLE
use boot and store nodes from `shards.test`

### PR DESCRIPTION
**Pull request by felicio**

### CODE REVIEW

**Comment by felicio** [comment]
relates to https://github.com/status-im/status-web/issues/520

**Comment by jrainville** [comment]
@felicio what is the state of this PR? When it's ready, we're gonna add the Status community to the curation directory

File: packages/DApp/src/providers/waku/provider.tsx
Comment (felicio) [inline packages/DApp/src/providers/waku/provider.tsx:1]: note: @status-im/web here I think server actions or api routes make sense for the client, but would require migrating from webpack to next. Something I'd vote for doing in another PR.

Comment (prichodko) [reply packages/DApp/src/providers/waku/provider.tsx:1]: why does the dapp need server?

**Review by prichodko — APPROVED** [review]

**Review by osmaczko — APPROVED** [review]

File: packages/DApp/src/providers/waku/provider.tsx
Comment (felicio) [reply packages/DApp/src/providers/waku/provider.tsx:1]: I raised it in relation to the connection limit https://grafana.infra.status.im/goto/uaHpGGJSz?orgId=1.

**Pull request merged**